ui/src/app/apps/apps.service.ts

```diff
@@ -109,10 +109,12 @@ export class AppsService {
     }
     params.append('force', appRegistration.force ? 'true' : 'false');
 
-    const options = new RequestOptions({ headers: headers, params: params });
+    const options = HttpUtils.getDefaultRequestOptions();
+    options.params = params;
+
     console.log(options.params);
 
-    return this.http.post(AppsService.appsUrl + '/' + appRegistration.type + '/' + appRegistration.name, {}, options)
+    return this.http.post(AppsService.appsUrl + '/' + ApplicationType[appRegistration.type] + '/' + appRegistration.name, {}, options)
```

Comment:
Just curious of why we are doing an enum lookup, when the appRegistration.type is already an ApplicationType.